build.sbt

```diff
@@ -14,6 +14,33 @@ licenses := Seq("Apache-2.0" -> url("http://www.apache.org/licenses/LICENSE-2.0"
 
 startYear := Some(2018)
 
+ThisBuild / developers := List(
+  Developer(
+    id = "eed3si9n",
+    name = "Eugene Yokota",
+    email = "eed3si9n@gmail.com",
+    url = url("https://github.com/eed3si9n")
+  ),
+  Developer(
+    id = "eed3si9n",
+    name = "Eugene Yokota",
+    email = "eed3si9n@gmail.com",
+    url = url("https://github.com/eed3si9n")
+  ),
```

Comment:
You have Eugene twice in the list. Also not sure if he wants his e-mail published here (email can stay empty and publishing will still work).

Also I have seen various styles for this developers fields...
Alternative would be to put in something very generic like:
https://github.com/sbt/sbt-multi-jvm/blob/main/build.sbt#L45-L50